test/cql-exports-test.ts

```diff
@@ -28,5 +29,5 @@ describe('CQL Exports', () =>
       it(`should export ${name}`, () => {
         cql[name].should.be.Function();
         cql[name].name.should.equal(name);
-      }))(libName)
+      }))(libName as keyof typeof cql)
```

Comment:
I am not sure I would have come up with this `as keyof typeof` magic.  You must be a sorcerer.